prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -120,7 +119,7 @@ class BatchFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine(
                             outReport.id,
                             actionHistory.generatingEmptyReport
                         )
-                        workflowEngine.dispatchReport(outEvent, outReport, actionHistory, receiver, txn, null)
+                        workflowEngine.dispatchReport(outEvent, outReport, actionHistory, receiver, txn)
```

Comment:
The majority of the changes here are actually removing references to the Azure `context` variable that was being passed all over the place (like this null param here), just for logging.

Since I was tweaking the logging, this seemed appropriate.

Uses of the old `context` have been replaced by uses of the Logging interface, which is our standard.   Not done everywhere, but we're inching closer.